doc/news/changes/minor/20210318MaximilianBergbauer

```diff
@@ -0,0 +1,3 @@
+New: Adds a new generalized MPI_Bcast function for arbitrary data types T.
```

Comment:
Could you mention the actual name of the function here? This way doxygen will place a hyperlink to the function in the changelog.